src/components/CreateSighting.scss

```diff
@@ -0,0 +1,112 @@
+@import "../App.scss";
+
+@media only screen and (max-width: 430px){
+    h1.sighting-form{
+        font-size:40px;
+    }
+
+    form{
+        background: $darker-background;
+        position:relative;
+        padding:20px;
+        height:380px;
+    }
```

Comment:
I'm wary of styling for elements using the tag name like this as it will apply to all form elements across the site. It would be better to use a class here